documentation/src/test/java/org/hibernate/search/documentation/gettingstarted/withhsearch/withoutanalysis/GettingStartedWithoutAnalysisIT.java

```diff
@@ -100,11 +101,11 @@ public void test() {
 		OrmUtils.withinJPATransaction( entityManagerFactory, entityManager -> {
 			// tag::searching-objects[]
 			// Not shown: get the entity manager and open a transaction
-			SearchSession searchSession = Search.session( entityManager ); // <1>
+			SearchMapping searchMapping = Search.mapping( entityManager.getEntityManagerFactory() ); // <1>
 
-			SearchScope<Book> scope = searchSession.scope( Book.class ); // <2>
+			SearchScope<Book> scope = searchMapping.scope( Book.class ); // <2>
 
-			SearchResult<Book> result = scope.search() // <3>
+			SearchResult<Book> result = scope.search( entityManager ) // <3>
```

Comment:
Is this mandatory? I must admit I'm not excited by the fact that we have to passe the EMF *and* the EM. It makes things far less fluid IMHO.

Reply:
Eventually, people who don't want to load entities will be allowed to call `scope.search()` (if using single-tenancy) or `scope.search(tenantId)` (if using multi-tenancy). That's not implemented yet.

For people who actually want to load entities, well, we need an entity manager. Yes, we have a factory, but I don't think creating the entity manager ourselves silently would be a good idea. The loaded entities would not be usable anyway.

I could provide a way to pass the scope to a `SearchSession`, e.g. `searchSession.search( scope )`, but then you need an entity manager to create the `SearchSession`, so it's just more verbose.

Also, please keep in mind this is just the syntax with scopes, which is not really the one I want to promote. Let's say it's an alternate, lower-level syntax. The first syntax, with a `SearchSession`, is much better in most cases.

Reply:
If the only problem you have is with the example itself, and the appearance of code for users who just don't want lambdas, I suppose I could also restore the `scope()` methods on the session. Then users will be able to trigger the search and create scope from a single object that was created from an entity manager.

However, the scopes will still be session-independent, so the `search()` method on those will still take an `EntityManager` in parameter. This method generally won't be useful though, since people can just call `search` on the session.